src/endpoints/auth/schemas/sign-up-or-in.schema.ts

```diff
@@ -1,9 +1,9 @@
-export interface ISignUpBodyInput {
+export interface ISignUpOrInBodyInput {
```

Comment:
Now I use types in such cases and I only use interfaces when I describe classes. But it's fine